### PR TITLE
(Release 3.14) Fix CTS (revert header change)

### PR DESCRIPTION
**Pull request by planetf1**
This reverts commit 7f460327a5a2bdd2da6722777ddcbc095cc83d92.

Signed-off-by: Nigel Jones <nigel.l.jones+git@gmail.com>

## Description

This reverts the final commit in 7f460327a5a2bdd2da6722777ddcbc095cc83d92 which was an optional cherry pick to fix 
a missing header version.

With this final fix CTS fails, however two remaining commits do fix CTS, and are needed for 3.14

I suggest persuing the header fix in 3.15/main only. The cts failures in the graph repo may be due to conflict resolution or the actual fix, but are not essential for 3.14

## Testing

Built a docker image without the final fix for header version (I reapplied rather than reverted, but end result should be same)
Ran graph CTS which passed with 0 errors

### CODE REVIEW

**Comment by planetf1** [comment]
Do not merge yet until retested

File: open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/connectors/stores/metadatacollectionstore/properties/instances/InstanceAuditHeader.java
Comment (planetf1) [reply open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/connectors/stores/metadatacollectionstore/properties/instances/InstanceAuditHeader.java:582]: @sonatype-lift ignoreall

**Review by lpalashevski — APPROVED** [review]

**Comment by planetf1** [comment]
A fix omitting the header change, passes CTS.

@lpalashevski suggest we merge this now & retest after merge+release pipeline

**Pull request merged**
